libs/SalesforceSDKCore/SalesforceSDKCoreTests/SFSmartStoreTests.m

```diff
@@ -40,27 +38,19 @@
 #import <SalesforceSecurity/SFPasscodeManager+Internal.h>
 #import <SalesforceSecurity/SFPasscodeProviderManager.h>
 #import "SFSecurityLockout+Internal.h"
-#import "SFUserAccountManager.h"
 #import <SalesforceSecurity/SFKeyStoreManager.h>
 #import <SalesforceSecurity/SFEncryptionKey.h>
 #import <SalesforceCommonUtils/NSString+SFAdditions.h>
 #import <SalesforceCommonUtils/NSData+SFAdditions.h>
 
-NSString * const kTestSmartStoreName   = @"testSmartStore";
-NSString * const kTestSoupName   = @"testSoup";
+#define kTestSmartStoreName  @"testSmartStore"
```

Comment:
Any particular reason to use ```#define``` instead of ```const```? Isn't it more optimal in ```iOS``` to use a ```const``` instead of a preprocessor macro, since it's limited to this file (I could be wrong in my assumption)? :-)